test_manifests/manifest.json

```diff
@@ -0,0 +1,11899 @@
+                                "@type": "dctypes:Image",
+                                "format": "image/jpeg",
+                                "service": {
+                                    "@context": "http://iiif.io/api/image/2/context.json",
+                                    "@id": "https://iiif.sinaimanuscripts.library.ucla.edu/iiif/2/ark%3A%2F21198%2Fz1fv04p5%2Fvg02tj78",
+                                    "profile": "http://iiif.io/api/image/2/level2.json"
+                                }
+                            }
+                        }
+                    ]
+                },
+                {
+                    "@type": "sc:Canvas",
+                    "label": "f. 084r",
+                    "@id": "https://iiif.library.ucla.edu/ark%3A%2F21198%2Fz1fv04p5/manifest/canvas/nt67kv7b",
+                    "width": 7030,
+                    "height": 8732,
+                    "images": [
+                        {
+                            "@type": "oa:Annotation",
+                            "@id": "https://iiif.library.ucla.edu/ark%3A%2F21198%2Fz1fv04p5/annotation/nt67kv7b",
+                            "motivation": "sc:painting",
+                            "on": "https://iiif.library.ucla.edu/ark%3A%2F21198%2Fz1fv04p5/manifest/canvas/nt67kv7b",
+                            "resource": {
+                                "@id": "https://iiif.sinaimanuscripts.library.ucla.edu/iiif/2/ark%3A%2F21198%2Fz1fv04p5%2Fnt67kv7b/full/600,/0/default.jpg",
+                                "@type": "dctypes:Image",
+                                "format": "image/jpeg",
+                                "service": {
+                                    "@context": "http://iiif.io/api/image/2/context.json",
+                                    "@id": "https://iiif.sinaimanuscripts.library.ucla.edu/iiif/2/ark%3A%2F21198%2Fz1fv04p5%2Fnt67kv7b",
+                                    "profile": "http://iiif.io/api/image/2/level2.json"
+                                }
+                            }
+                        }
+                    ]
+                },
+                {
+                    "@type": "sc:Canvas",
+                    "label": "f. 084v",
+                    "@id": "https://iiif.library.ucla.edu/ark%3A%2F21198%2Fz1fv04p5/manifest/canvas/c701hf62",
+                    "width": 7033,
+                    "height": 8729,
+                    "images": [
+                        {
+                            "@type": "oa:Annotation",
+                            "@id": "https://iiif.library.ucla.edu/ark%3A%2F21198%2Fz1fv04p5/annotation/c701hf62",
+                            "motivation": "sc:painting",
+                            "on": "https://iiif.library.ucla.edu/ark%3A%2F21198%2Fz1fv04p5/manifest/canvas/c701hf62",
+                            "resource": {
+                                "@id": "https://iiif.sinaimanuscripts.library.ucla.edu/iiif/2/ark%3A%
```

Comment:
instead of a range, this should use the `canvases` property and list all the canvases in this range. For this test, you can just list the first 10 or so.

Follow this model for the remaining 2 ranges (again, you can just use the first 10 canvases since this is a test)